src/graph_compute/chunk_scheduling.h

```diff
@@ -28,6 +28,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
Oh, that's chunk_scheduling.h, not phase_scheduling.h.  Sorry.  chunk_scheduling.h is essentially abandoned, as it is totally dominated by phase_scheduling.h.  In fact, I think if you just set NUM_PHASES=1, it becomes chunk_scheduling.h.  I'm not sure about the chunk_scheduling.h logic, the logic in phase_scheduling.h looks correct to me.